schema.graphql

```diff
@@ -39,6 +45,8 @@ type PrizePool @entity {
 
   reserveFeeControlledToken: Bytes!
 
+  sablierStreamId: SablierStream @derivedFrom(field: "prizePool")
```

Comment:
You should simply call this `sablierStream` since it's referring to an entity (not a basic type)